spec/models/dealership_spec.rb

```diff
@@ -23,5 +23,19 @@
         expect(Dealership.default_scope).to eq ([ted, harrison, rex])
       end
     end
+
+    describe '.alphabetize' do
```

Comment:
Great, it's important that we're 100% covered on model tests, looks like everything's covered here.